media/system/js/tabs-state.js

```diff
@@ -8,8 +8,9 @@
  * keeping state in sessionStorage with better handling of multiple tab widgets per page
  * and not saving the state if there is no id in the url (like on the CREATE page of content)
  */
+Joomla = window.Joomla || {};
 
-jQuery(function ($) {
+(function(Joomla) {
 
 	// Ensure in IE8 we can use xpath
 	if (typeof wgxpath.install === "function") {
```

Comment:
IE 11 is not compatible with https://caniuse.com/#search=Xpath
Can you check that this script is working/not working with out the polyfill?